docs/source/how_to_guides/configure_cloud_storage_cred.md

```diff
@@ -0,0 +1,101 @@
+# Configure Cloud Storage Credentials
+
+Streaming dataset supports the following cloud storage providers to stream your data directly to your instance.
+- [Amazon S3](https://aws.amazon.com/s3/)
+- [Google Cloud Storage](https://cloud.google.com/storage)
+- [Oracle Cloud Storage](https://www.oracle.com/cloud/storage/)
+
+## Amazon S3
+
+For S3 bucket with a public access, no additional setup is required, simply specify the S3 URI of the resource.
+
+### MosaicML platform
+
+For [MosaicML platform](https://www.mosaicml.com/cloud) users, follow the steps mentioned in the [Amazon S3](https://mcli.docs.mosaicml.com/en/latest/secrets/s3.html) doc on how to configure the cloud provider credentials.
```

Comment:
Should match the MCLI docs header so [Amazon S3] -> [AWS S3]

Reply:
After the link put MCLI so that it reads **MCLI doc** on how to configure...

Reply:
Fixed in https://github.com/mosaicml/streaming/pull/169/commits/c164c1329e2a1cdfc5d49ece7647e05bca8ce411.